docs/04-integrating-with-sygma/01-Sygma Widget/03-react.md

```diff
@@ -12,6 +12,7 @@ Add the following dependencies to integrate the widget into any React project:
 
 ```bash
 yarn add @polkadot/extension-inject
+yarn add @polkadot/types
```

Comment:
this is a dev dependency, it should be `yarn add -D`. Also I wonder if there is a warning that you receive that makes you install this dep

Reply:
updated. 

got the @polkadot/types error... it was actually when i want to configure environment to TESTNET.
my apologies... so this shouldn't actually be a dependency to be added here, correct? especially since it's just "quick setup" docs essentially

```
✘ [ERROR] Could not resolve "@polkadot/types/create"

    .yarn/cache/@buildwithsygma-sygma-sdk-core-npm-2.10.0-4f142153bf-a463a41449.zip/node_modules/@buildwithsygma/sygma-sdk-core/dist-esm/chains/EVM/helpers.js:2:29:
      2 │ import { TypeRegistry } from '@polkadot/types/create';
        ╵                              ~~~~~~~~~~~~~~~~~~~~~~~~

  The Yarn Plug'n'Play manifest forbids importing "@polkadot/types" here because it's not listed as
  a dependency of this package:

    .pnp.cjs:437:33:
      437 │           "packageDependencies": [\
          ╵                                  ~~

  You can mark the path "@polkadot/types/create" as external to exclude it from the bundle, which
  will remove this error and leave the unresolved path in the bundle.

9:57:59 p.m. [vite] error while updating dependencies:
Error: Build failed with 1 error:
.yarn/cache/@buildwithsygma-sygma-sdk-core-npm-2.10.0-4f142153bf-a463a41449.zip/node_modules/@buildwithsygma/sygma-sdk-core/dist-esm/chains/EVM/helpers.js:2:29: ERROR: Could not resolve "@polkadot/types/create"
    at failureErrorWithLog (/Users/timho/my-dapp/.yarn/unplugged/esbuild-npm-0.21.5-d85dfbc965/node_modules/esbuild/lib/main.js:1472:15)
    at /Users/timho/my-dapp/.yarn/unplugged/esbuild-npm-0.21.5-d85dfbc965/node_modules/esbuild/lib/main.js:945:25
    at /Users/timho/my-dapp/.yarn/unplugged/esbuild-npm-0.21.5-d85dfbc965/node_modules/esbuild/lib/main.js:1353:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Reply:
at the same time, im assuming `yarn add @polkadot/extension-inject` gives the interface the ability to inject polkadot wallets. so if we're talking about base packages that need to be added, then perhaps we keep both /extension-inject AND /types as dependencies in the docs... especially if we want devs to play on testnet.

Reply:
so if I get this correctly, once you pass env testnet, the widget is giving you this error?

Reply:
the extension-inject is actually a dep required -if not mistaken- by the SDK

Reply:
> so if I get this correctly, once you pass env testnet, the widget is giving you this error?

correct

Reply:
> the extension-inject is actually a dep required -if not mistaken- by the SDK

yea, it's interesting, i just re-ran through the docs again and for sure polkadot/types is needed for environment configuration. on the other hand, extension-inject is not required and i can still connect polkadot.js to my react app. weird.